src/utils/exceptionThrowers.ts

```diff
@@ -36,7 +37,15 @@ export const throwChatSDKError = (chatSDKError: ChatSDKErrors, e: unknown, scena
         console.error(message);
     }
 
-    throw new Error(exceptionDetails.response);
+    const errorObject = {
+        message: exceptionDetails.response
+    };
+    
+    if ((e as any)?.response?.status) {
+        (errorObject as any).status = (e as any).response.status;

```

Comment:
I'd suggest to have `HTTPResponseStatusCode`. `status` is too generic and could mean a lot of different things

Reply:
changed